app/src/main/java/com/example/bookclub/SearchActivity.java

```diff
@@ -156,11 +157,20 @@ public boolean onOptionsItemSelected(MenuItem item) {
             Toast.makeText(SearchActivity.this, "Logout Successfully!", Toast.LENGTH_SHORT).show();
             onLogout();
             return true;
+        }else if(id == R.id.action_library){
+            Toast.makeText(SearchActivity.this, "now in library", Toast.LENGTH_SHORT).show();
+            onLibrary();
+            return true;
         }
 
         return super.onOptionsItemSelected(item);
     }
 
+    private void onLibrary() {
```

Comment:
These methods can be called onClickLibrary and onClickLogout so it is clear that they are user actions.